SpatialGDK/Source/SpatialGDK/Private/LoadBalancing/GridBasedLBStrategy.cpp

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "LoadBalancing/GridBasedLBStrategy.h"
+
+#include "EngineClasses/SpatialNetDriver.h"
+#include "Utils/SpatialActorUtils.h"
+
+UGridBasedLBStrategy::UGridBasedLBStrategy()
+	: Super()
+	, Rows(1)
+	, Cols(1)
+	, WorldWidth(10000.f)
+	, WorldHeight(10000.f)
```

Comment:
This seems a bit arbitrary. Would you consider using constants for consistency with configuration defaults such as https://github.com/spatialos/UnrealGDK/blob/master/SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSettings.h#L23 and https://github.com/spatialos/UnrealGDK/blob/master/SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSettings.h#L118?